hkmc2/shared/src/test/mlscript/ucs/hygiene/HygienicBindings.mls

```diff
@@ -45,7 +39,8 @@ fun h1(a) =
     a is None then 0
 
 // FIXME
```

Comment:
Is this still a FIXME?

Reply:
No. It's no longer a FIXME. I've clarified the comments.